src/metabase/db/custom_migrations.clj

```diff
@@ -1070,6 +1070,20 @@
                                            :from   [:revision]
                                            :where  [:= :model "Card"]})))))
 
+(define-migration DeleteScanFieldValuesTriggerForDBThatTurnItOff
```

Comment:
I think this migration deserves a docstring explaining the issue it fixes.

Reply:
agree, I add oned here: [6f842fb](https://github.com/metabase/metabase/pull/41348/commits/6f842fb4a305d2f8cd54ddba86678b5873996023)